cluster/images/hyperkube/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM google/debian:wheezy
```

Comment:
we sadly don't have a backports wheezy docker image:

https://registry.hub.docker.com/u/google/debian/tags/manage/

We probably should...